content/en/monitors/guide/how-to-set-up-rbac-for-monitors.md

```diff
@@ -0,0 +1,216 @@
+                        ...
+                    ]
+                }
+            }
+        },
+        ...
+    ]
+}
+```
+
+Use the [Create][2] or [Edit a monitor][3] API endpoint and the `restricted_roles` parameter to restrict monitor edition to a specific set of roles and to the monitor's creator. 
+
+**Note:** You can specify one or multiple role UUIDs. Setting `restricted_roles` to `null` allows monitor edition for all users with [Monitor Write permissions][4].
+
+```bash
+curl --location --request POST 'https://api.datadoghq.com/api/v1/monitor' \
+--header 'Content-Type: application/json' \
+--header 'DD-API-KEY: <DD-API-KEY>' \
+--header 'DD-APPLICATION-KEY: <DD-APPLICATION-KEY>' \
+--data-raw '{
+  "message": "You may need to add web hosts if this is consistently high.",
+  "name": "Bytes received on host0",
+  "options": {
+    "no_data_timeframe": 20,
+    "notify_no_data": true
+  },
+  "query": "avg(last_5m):sum:system.net.bytes_rcvd{host:host0} \u003e 100",
+  "tags": [
+    "app:webserver",
+    "frontend"
+  ],
+  "type": "query alert",
+  "restricted_roles": ["89f5dh86-e470-11f8-e26f-4h656a27d9cc"]
+}'
+```
+
+For more information, see [Roles][5] and [Monitors API documentation][6] .
+
+
+[1]: /api/latest/roles/#list-roles
+[2]: /api/latest/monitors/#create-a-monitor
+[3]: /api/latest/monitors/#edit-a-monitor
+[4]: /account_management/rbac/permissions/#monitors
+[5]: /api/latest/roles/
+[6]: /api/latest/monitors/
+{{% /tab %}}
+{{< /tabs >}}
+
+## Deprecated locked mechanism
+
+Previous to the above described role restriction option, monitors could be locked, meaning only their creator and users with the [Datadog Admin Role][2] would be able to edit them. 
+
+{{< img src="/monitors/guide/monitor_rbac_locked.jpg" alt="RBAC Locked Monitor" style="width:70%;">}}
+
+This locked mechanism was deprecated and it is now recommended to use the role restriction option, which gives you more flexibility to define users allowed to edit monitors.
+
+You might however have a number of existing monitors still leveraging the locked mechanism under the hood. This mechanism is still supported and respected for now, which means locked monitors should not be editable by users who do not have the [Datadog Admin Role][2] attached or are not the monitor's creator.
+
+Below sections describe how the change from the old locked mechanism to the new role restriction one is performed, depending on the way you manage your monitors.
+
+### API
+
+Although deprecated, the `locked` parameter corresponding to the above mentioned locked mechanism is still supported. This means you can progressively update the definition of your monitors managed through API/Terraform to stop using `locked` and start using `restricted_roles` (parameter attached with the new role restriction option).
+
+For more information on how to update your monitors' definitions, see the [Edit a monitor API endpoint documentation][3] and the dedicated [Monitor API Options guide][4].
```

Comment:
```suggestion
For more information on how to update your monitors' definitions, see the [Edit a monitor API endpoint][3] and the [Monitor API Options guide][4].
```

Reply:
> see the Edit a monitor API endpoint

Doesn't that sound weird? What about "see Edit a monitor API endpoint"? (It might not sound weird for an English speaker though! It felt weird to me so wanted to flag but let me know if it is actually ok!)

> Monitor API Options guide

Based on your other comments, it looks like we are trying to move away from saying "see XYZ **documentation**", why is it ok to mention guide here?

Reply:
```suggestion
For more information on how to update your monitors' definitions, see [Edit a monitor API endpoint][3] and the [Monitor API Options][4].
```
"Guide" is more specific than "documentation". Sorry, it's hard to remember to be consistent across a whole review. But you're right, to be consistent with the style guide, it's better to remove "Guide" also.

I left in the "the" by accident.

How about the phrasing above?

Reply:
LGTM, committing!